src/authV2/azext_authV2/_params.py

```diff
@@ -76,6 +78,10 @@ def load_arguments(self, _):
         c.argument('allowed_token_audiences', options_list=['--allowed-token-audiences', '--allowed-audiences'],
                    help='The configuration settings of the allowed list of audiences from which to validate the JWT token.')
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+        c.argument('tenant_id', options_list=['--tenant-id'],
+                   help='The tenant id of the application.')
+        c.argument('cloud', options_list=['--cloud'], arg_type=get_enum_type(CLOUD_NAMES),
```

Comment:
What is the purpose of this? A user needs to switch the cloud used before they can use the command groups, so the default cloud being used should give you the name of the cloud already?  https://docs.microsoft.com/en-us/cli/azure/manage-clouds-azure-cli